examples/e2e/consumer.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const request = require('superagent-bluebird-promise');
```

Comment:
you don't need that... Superagent 2 already has promises embedded into it. Unless you want to use something specific from Bluebird this is overkill